### DISCUSSION TITLE
A note of thanks!

### DISCUSSION THREAD

**Discussion by danielwestendorf** [skroutz/rspecq#46] [Show and tell]
I just wanted to share the success I've had with this gem on a real world application.

Test suite is heavy on browser based testing. 20 worker nodes. ~1min of CI execution time is spent doing setup (installing ruby, dep installs). Prior I was using the `parallel_tests` gem.

Execution time before: 5m 2s
Execution time after: 2m 36s
🚀 🏎 🎉 

Running on CircleCI w/ a redis instance from a cloud provider.

```yaml
      - run: |
          bundle exec rspecq --build=$CIRCLE_BUILD_NUM \
                             --worker=$CIRCLE_NODE_INDEX \
                             --update-timings \
                             --file-split-threshold=5 \
                             spec/

      - run: |
          bundle exec rspecq --build=$CIRCLE_BUILD_NUM --worker=$CIRCLE_NODE_INDEX --report
```

Thanks for the great project and accepted [PR](https://github.com/skroutz/rspecq/issues/37) to support redis URLs! 💙

**Comment by agis** [comment]
Hey @danielwestendorf, that's super encouraging to hear! Thanks for the kind words and your contributions 🙌🏾 💯
